app/gui/view/graph-editor/src/lib.rs

```diff
@@ -1762,25 +1762,25 @@ impl GraphEditorModelWithNetwork {
 #[derive(Debug, Clone, CloneRef)]
 #[allow(missing_docs)] // FIXME[everyone] Public-facing API should be documented.
 pub struct GraphEditorModel {
-    pub display_object:      display::object::Instance,
-    pub app:                 Application,
-    pub breadcrumbs:         component::Breadcrumbs,
-    pub cursor:              cursor::Cursor,
-    pub nodes:               Nodes,
-    pub edges:               Edges,
-    pub vis_registry:        visualization::Registry,
-    pub drop_manager:        ensogl_drop_manager::Manager,
-    pub navigator:           Navigator,
-    pub add_node_button:     Rc<component::add_node_button::AddNodeButton>,
-    tooltip:                 Tooltip,
-    touch_state:             TouchState,
-    visualisations:          Visualisations,
-    frp:                     Frp,
-    profiling_statuses:      profiling::Statuses,
-    profiling_button:        component::profiling::Button,
-    styles_frp:              StyleWatchFrp,
-    selection_controller:    selection::Controller,
-    execution_mode_selector: execution_mode_selector::ExecutionModeSelector,
+    pub display_object: display::object::Instance,
+    pub app: Application,
+    pub breadcrumbs: component::Breadcrumbs,
+    pub cursor: cursor::Cursor,
+    pub nodes: Nodes,
+    pub edges: Edges,
+    pub vis_registry: visualization::Registry,
+    pub drop_manager: ensogl_drop_manager::Manager,
+    pub navigator: Navigator,
+    pub add_node_button: Rc<component::add_node_button::AddNodeButton>,
+    tooltip: Tooltip,
+    touch_state: TouchState,
+    visualisations: Visualisations,
+    frp: Frp,
+    profiling_statuses: profiling::Statuses,
+    profiling_button: component::profiling::Button,
+    styles_frp: StyleWatchFrp,
+    selection_controller: selection::Controller,
+    execution_environment_selector: execution_environment_selector::ExecutionEnvironmentSelector,
```

Comment:
I would import `ExecutionEnvironmentSelector`, as the module name is just repeated information.